Cargo.toml

```diff
@@ -67,9 +68,9 @@ fs_extra = "1.1.0"
 predicates = "1.0.5"
 
 [features]
-# We build Linux binaries with vendored openssl
-# to avoid SSL issues
-vendored-openssl = ['openssl/vendored']
```

Comment:
we should probably keep the `vendored-openssl` feature flag around so that if people have CI set up to install from cargo with that feature it doesn't just start failing